woocommerce/woocommerce-gateway-stripe
# feat: payment methods availability

# Changes proposed in this Pull Request:

Fixes https://github.com/woocommerce/woocommerce-gateway-stripe/issues/1811

Ensures that we can persist the selected payment methods.
Data is persisted in the existing controller.

![2021-09-17 09 48 08](https://user-images.githubusercontent.com/273592/133803008-57ca5cd0-0a13-4d60-bd63-d5f1bc7995e8.gif)


# Testing instructions

- Go to http://localhost:8082/wp-admin/admin.php?page=wc-settings&tab=checkout&section=stripe
- Ensure the `capture` setting is set to `no` (if you ever set it - but I don't think you did)
- Go to http://localhost:8082/wp-admin/admin.php?page=wc_stripe-onboarding_wizard
- Check/uncheck payment methods checkboxes
- Click the "Add payment methods" button
- Payment methods are persisted
- Refresh the page
- Previously saved settings are displayed
